Makefile

```diff
@@ -12,12 +15,29 @@ create_database:
 clean:
 	docker ps -a -q | xargs docker kill;docker ps -a -q | xargs docker rm
 
+down:
+	docker-compose down
+
 bundle:
 	docker-compose run server bin/bundle-extensions
 
 tests:
 	docker-compose run server tests
 
+lint:
+	./flake8_tests.sh
+
+backend-unit-tests: up test_db
```

Comment:
Won't this try to run `test_db` everytime and hence fail on future tries?

Reply:
Yes, it is not idempotent. 🤔 
Ho about *test_db* recreating the database tests?